### PR TITLE
data analyst

### PR DESCRIPTION
**Pull request by HaniffZaid**
try pull request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,40 @@
...
+<a name="readme-top"></a>
+
+  <h1 align="center"><a href="https://github.com/HaniffZaid/KoNoHa-Property.git">
+    <img src="KONOHAgold.png" alt="Logo" width="400" height="264">
+  </a>
+
+  HOUSE PRICE PREDICTION IN KUALA LUMPUR</h1>
+   <p align="center">
+    Produced by KONOHA Holdings Bhd
+  </p>   <p align="center">
```
Comment (kameerahariru) [inline README.md]: taknak konoha, nak nama lain

File: README.md [hunk 2]
```diff
@@ -1 +1,40 @@
...
+    Kamilla Halil, Nasharuddin Ghazali & Haniff Zaid
+  </p>
+
+  <!-- TABLE OF CONTENTS -->
+<details>
+  <summary>Table of Contents</summary>
+  <ol>
+    <li>
+      <a href="#about-the-project">About The Project</a>
+      <ul>
```
Comment (kameerahariru) [inline README.md]: blablabla

File: README.md [hunk 3]
```diff
@@ -1 +1,40 @@
...
+    </li>
+    <li><a href="#usage">Usage</a></li>
+    <li><a href="#contact">Contact</a></li>
+    <li><a href="#acknowledgement">Acknowledgement</a></li>
+  </ol>
+</details>
+
+
+<!-- About The Projects -->
+## About The Project
```
Comment (kameerahariru) [inline README.md]: comment auto

**Review by kameerahariru — COMMENTED** [review]
Please review three comment that i made ASAP!

File: README.md [hunk 4]
```diff
@@ -1 +1,40 @@
-# Property-Listings-in-Kuala-Lumpur
\ No newline at end of file
+<a name="readme-top"></a>
+
+  <h1 align="center"><a href="https://github.com/HaniffZaid/KoNoHa-Property.git">
+    <img src="KONOHAgold.png" alt="Logo" width="400" height="264">
```
Comment (kameerahariru) [inline README.md]: dapat tengok tak nash

Comment (HaniffZaid) [reply README.md]: mana nash?

File: README.md (hunk 3 above)
Comment (HaniffZaid) [reply README.md]: nash nash nash

Comment (Nash887) [reply README.md]: dont givap

**Pull request closed without merging**